clang/lib/Parse/ParseDecl.cpp

```diff
@@ -6007,7 +6007,7 @@ bool Parser::isConstructorDeclarator(bool IsUnqualified, bool DeductionGuide,
 
   // A C++11 attribute here signals that we have a constructor, and is an
   // attribute on the first constructor parameter.
-  if (getLangOpts().CPlusPlus11 &&
+  if (getLangOpts().CPlusPlus &&
```

Comment:
Is this ever false? This functions looks to me like it would only ever be called if we're parsing C++.